src/types.rs

```diff
@@ -585,9 +445,9 @@ impl Default for RuntimeConfig {
 		RuntimeConfig {
 			http_server_host: "127.0.0.1".to_owned(),
 			http_server_port: (7000, 0),
-			ipfs_port: (37000, 0),
-			ipfs_seed: None,
-			ipfs_path: format!("avail_ipfs_node_{}", 1),
+			libp2p_port: (37000, 0),
+			libp2p_seed: None,
+			libp2p_psk_path: "./avail/psk".to_owned(),
```

Comment:
We should add the default path to the readme, for ease of use...

Reply:
this is just an example cfg
best to use an empty string, for now